File: Basics/Conditional-Statements-Advanced/Exercise/journey.py
budget = float(input())
season = input()
destination = ""
spent_money = 0
holiday_type = ""

if budget <= 100:
    destination = "Bulgaria"
    if season == "summer":
        spent_money = budget * 0.30
        holiday_type = "Camp"
    elif season == "winter":
        spent_money = budget * 0.70
        holiday_type = "Hotel"
elif budget <= 1000:
    destination = "Balkans"
    if season == "summer":
        spent_money = budget * 0.40
        holiday_type = "Camp"
    elif season == "winter":
        spent_money = budget * 0.80
        holiday_type = "Hotel"
elif budget > 1000:
    destination = "Europe"
    spent_money = budget * 0.90
    holiday_type = "Hotel"

print(f"Somewhere in {destination}\n"
      f"{holiday_type} - {spent_money:.2f}")

File: JavaScript/Basics/0.3-Conditional-Steatements-Advanced/Exercise/journey.js
function journey(input) {
    let budget = Number(input[0]);
    let season = input[1];
    let destination, shelterType, spendMoney;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season == "summer") {
            shelterType = "Camp";
            spendMoney = budget * 0.30;
        } else if (season == "winter") {
            shelterType = "Hotel";
            spendMoney = budget * 0.70;
        }
    } else if (budget <= 1000) {
        destination = "Balkans";
        if (season == "summer") {
            shelterType = "Camp";
            spendMoney = budget * 0.40;
        } else if (season == "winter") {
            shelterType = "Hotel";
            spendMoney = budget * 0.80;
        }
    } else if (budget > 1000) {
        destination = "Europe";
        shelterType = "Hotel";
        spendMoney = budget * 0.90;
    }

    console.log(`Somewhere in ${destination}\n${shelterType} - ${spendMoney.toFixed(2)}`);

}
